client/src/components/Data.js

```diff
@@ -9,6 +9,8 @@ import BlockHeader from "./BlockHeader";
 import PandidBoxed from "../images/pandidboxed.png";
 
 const VALVE_MAP = { 1: "OPEN", 0: "CLOSED" };
+const SENSOR_TEXT_PADDING = {"PT-5":{x: (420/784), y: 22/850}, "PT-P":{x: 635/784, y:262/850}, "PT-7":{x:101/784, y:385/850}, "PT-8":{x:548/784, y:757/850}};
```

Comment:
I feel like this should be moved to a different file, or there could be some better way to style the components